mautic/mautic
# Fix: API - GET tag replace Apostrophe with its HTML Entity Number

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✅ 
| New feature/enhancement? (use the a.x branch)      | ❌ 
| Deprecations?                          | ❌ 
| BC breaks? (use the c.x branch)        | ❌ 
| Automated tests included?              | ✅  
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



On GET API request for Tags containing Apostrophe(') in their name are replaced with it's HTML Entity Number(')

#### Steps to reproduce:
1. Open Postman or similar application
2. Create a GET request for tag (tag having ' in it)
3. Click on send
4. Verify the response (' replaced with ''' )

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. After PR steps to reproduce should work




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11116"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
